src/common/Platform.cpp

```diff
@@ -0,0 +1,6 @@
+#include "common/Platform.h"
+
+uint32_t Common::Platform::NetworkByteOrderLong(uint32_t value)
+{
+    return htonl(value);
```

Comment:
Didn't realize the winsock api had the same exact signature of the function as POSIX. That makes it easier.